osf_tests/conftest.py

```diff
@@ -11,27 +11,35 @@
 from website.project.views.contributor import notify_added_contributor
 
 # Silence some 3rd-party logging and some "loud" internal loggers
-SILENT_LOGGERS = [
-    'api.caching.tasks',
-    'factory.generate',
-    'factory.containers',
-    'framework.analytics',
-    'framework.auth.core',
-    'framework.celery_tasks.signals',
-    'website.app',
-    'website.archiver.tasks',
-    'website.mails',
-    'website.notifications.listeners',
-    'website.search.elastic_search',
-    'website.search_migration.migrate',
-    'website.util.paths',
-    'requests_oauthlib.oauth2_session',
-    'raven.base.Client',
-    'raven.contrib.django.client.DjangoClient',
-]
-for logger_name in SILENT_LOGGERS:
-    logging.getLogger(logger_name).setLevel(logging.CRITICAL)
+# SILENT_LOGGERS = [
+#     'api.caching.tasks',
+#     'factory.generate',
+#     'factory.containers',
+#     'framework.analytics',
+#     'framework.auth.core',
+#     'framework.celery_tasks.signals',
+#     'website.app',
+#     'website.archiver.tasks',
+#     'website.mails',
+#     'website.notifications.listeners',
+#     'website.search.elastic_search',
+#     'website.search_migration.migrate',
+#     'website.util.paths',
+#     'requests_oauthlib.oauth2_session',
+#     'raven.base.Client',
+#     'raven.contrib.django.client.DjangoClient',
+#     'osf.migrations.0001_initial',
+#     'osf.models',
+#     'addons.osfstorage.models',
+#     'rest_framework.pagination',
+#     'tests.base',
+#     'website.project',
+#     'django.db.models.fields',
+# ]
+# for logger_name in SILENT_LOGGERS:
+#     logging.getLogger(logger_name).setLevel(logging.CRITICAL)
 
+logging.disable(logging.WARNING)
```

Comment:
We don't want to disable *all* warnings. Some of these are useful. For example, we would not have found https://openscience.atlassian.net/browse/OSF-8136 if not for the warnings from Django.

We only want to disable *expected* warnings, which is why we had the list of `SILENT_LOGGERS`. Would it make sense to revert this change? I think your configuration change in `pytest.ini` looks correct and may be enough to close the ticket.